### PR TITLE
Remove wrong debug endpoint from docs

### LINKED ISSUE #2421: debug API endpoint doesn't work anymore
## Problem Definition

I tried to querry `/api/(version)/_debug/raiden_events/tokens/(token_address)` and got an invalid API endpoint response.
Is it supposed to be like that? If so --> remove the endpoint from the docu

### System Description
Used the latest *.tar [binary release](https://github.com/raiden-network/raiden/releases/download/v0.8.0/raiden-v0.8.0-linux.tar.gz)(v0.8.0) 
Python3.6, Ubuntu 18.04

### PR DESCRIPTION
**Pull request by LefterisJP**
Fix #2421

### CODE REVIEW

File: docs/rest_api.rst [hunk 1]
```diff
@@ -781,15 +781,15 @@ For ``raiden_events`` you can provide a ``limit`` and an ``offset`` number which
 
 Now for the internal Raiden events:
 
-.. http:get:: /api/(version)/_debug/raiden_events/tokens/(token_address)
+.. http:get:: /api/(version)/_debug/raiden_events/
```
Comment (hackaugusto) [inline docs/rest_api.rst]: I think all `_debug` endpoints should be removed from the docs.

Comment (LefterisJP) [reply docs/rest_api.rst]: Hmm it's good to mention at least which debug endpoints exist.

Comment (hackaugusto) [reply docs/rest_api.rst]: why? I thought these end points where for debugging, IOW for us to test some stuff.

Comment (LefterisJP) [reply docs/rest_api.rst]: For us and for any users we may need to ask to run these endpoints and provide us with the results. It's nice to have all the existing endpoints documented. The `_debug` endpoints could just have their existence documented for completeness sake and we can remove the examples.

I just want our docs to list all endpoints.

**Pull request merged**